keda/templates/manager/deployment.yaml

```diff
@@ -187,6 +187,13 @@ spec:
             {{- else }}
             {{- toYaml .Values.resources | nindent 12 }}
             {{- end }}
+        {{- if .Values.operator.extraContainers }}
+        {{- toYaml .Values.operator.extraContainers | nindent 8 }}
+        {{- end }}
+      {{- if .Values.operator.extraInitContainers }}
+      initContainers:
+      {{- toYaml .Values.operator.extraInitContainers | nindent 6 }}
```

Comment:
is 6 the correct value? containers and initContainers are at the same level, and you've set 8 for extra containers and 6 for extra initContainers. I guess that one of them is wrong

Reply:
@JorTurFer Fair! There's a bit of inconsistency because the `containers:` list is indented more than the `volumes:` list. I've now set it to 8 to make it consistent with containers